app/models/reports/region_summary.rb

```diff
@@ -71,9 +81,15 @@ def call
     end
 
     def for_regions
-      FacilityState
+      query = FacilityState
         .where(id_field => regions.map(&:id))
-        .where("cumulative_registrations IS NOT NULL OR cumulative_assigned_patients IS NOT NULL OR monthly_follow_ups IS NOT NULL")
+
+      filter = "cumulative_registrations IS NOT NULL OR cumulative_assigned_patients IS NOT NULL OR monthly_follow_ups IS NOT NULL"
+      if regions.any?(&:diabetes_management_enabled?)
+        filter += " OR cumulative_diabetes_registrations IS NOT NULL OR cumulative_assigned_diabetic_patients IS NOT NULL OR monthly_diabetes_follow_ups IS NOT NULL"
+      end
+
+      query.where(filter)
```

Comment:
All of this should go away. Currently cumulative_registrations only counts htn patients. We have created a separate story to introduce `cumulative_htn_registraions` and `cumulative_registrations` will count both htn and diabetes patients. 

We didn't do this as a part of this PR, since it'll cause us to rename this across the codebase, and explore metabase and fix appropriate questions. We think we can do this after the diabetes reports are done, so we don't have to block the release.

Reply:
I tried out the `or` and `merge` above which seems to work. Nonetheless, seems ok if we are getting rid of it in this epic.